code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -17,6 +19,11 @@
 	///Used to track if anything of note has happen while running process_atmos()
 	var/excited = TRUE
 
+	/// Max amount of heat allowed inside the machine before it starts to melt. -1 is special value meaning we wont check.
+	var/temp_limit = 5000
+	/// Max amount of pressure allowed inside of the canister before it starts to break. -1 is special value meaning we wont check.
+	var/pressure_limit = 50000
```

Comment:
we won't check vs we are immune

Reply:
also is it -1 or 0

Reply:
-1 earlier but moved it to 0. Forgot to update.